scripts/install

```diff
@@ -147,11 +147,16 @@ exec_occ() {
 # Set write access for the following commands
 chown -R $app: "$final_path" "$datadir"
 
+# Define password in an intermediate var
+# The fact that it's called _password allows it to be
+# picked up by Yunohost's auto-redact mecanism
+admin_password="$(ynh_string_random --length=6)"
```

Comment:
By the way length=6 ... that doesn't sounds super secure x_x

Reply:
Yes, but we remove this user later: https://github.com/YunoHost-Apps/nextcloud_ynh/blob/41d2c377623d58862c8ee3dc2dc70f1f8351fe4f/scripts/install#L258

Reply:
Oh I see

So less spooky than I tought, nvm